doslib/datatype.def

```diff
@@ -45,6 +45,32 @@ class: EnemyStats
     drop_chance, 8
     padding, 8[3]
 
+class: EnemyName
+    namePtr, 32
+    
+class: EnemyGraphics
+    tileData, 32
+    palette, 32
+    tileArrangement, 32
+    
+class: EnemyScript
+    spell_chance, 8
+    ability_chance, 8
+    spell_1, 8
```

Comment:
This should probably be: `spell, 8[8]` (or [9]?) so it gets generated as a proper Python array.